src/libraries/System.Runtime.InteropServices/src/System/Runtime/InteropServices/Marshalling/IIUnknownCacheStrategy.cs

```diff
@@ -12,10 +12,22 @@ namespace System.Runtime.InteropServices.Marshalling
     [CLSCompliant(false)]
     public unsafe interface IIUnknownCacheStrategy
     {
+        /// <summary>
+        /// A structure that contains information about a COM interface and its virtual method table.
+        /// </summary>
         public readonly struct TableInfo
         {
+            /// <summary>
+            /// The pointer to the <c>this</c> pointer to the unmanaged interface.
```

Comment:
Is this a pointer to the 'this' pointer or the pointer to the 'this' value?
```suggestion
            /// The <c>this</c> pointer of the unmanaged interface.
```